projects/train/configs/flow/cbc.yaml

```diff
@@ -20,6 +20,8 @@ model:
   class_path: train.models.flow.FlowModel
   init_args:
     outdir: ${oc.env:AMPLFI_OUTDIR}
+    # uncomment below when running `trainer.test` to load in a checkpoint
+    # checkpoint:
```

Comment:
This looks good. Is it possible to keep something like Null or None value instead of uncommenting?

Reply:
Ah yeah good call I can make it `null`

Reply:
done